src/waveform/renderers/allshader/waveformrendermark.cpp

```diff
@@ -1,20 +1,13 @@
 #include "waveform/renderers/allshader/waveformrendermark.h"
 
-#include <QDomNode>
 #include <QOpenGLTexture>
 #include <QPainterPath>
 
-#include "track/track.h"
-#include "util/color/color.h"
-#include "util/colorcomponents.h"
```

Comment:
It builds when I restore this include

Reply:
This is on Ubuntu 20.04, and it seems this and #12303 slip through because we don't have a 20.04 runner anymore even though we support it?